src/api/app/views/webui/request/beta_show_tabs/_overview.html.haml

```diff
@@ -1 +1,61 @@
-Overview
+.row
+  .col-md-8#overview
+    #description-text
+      - if bs_request.description.present?
```

Comment:
You should talk to @danidoni and see #12858 this is also included in the history now.

Reply:
talking has become solving merge conflicts 😉

Reply:
After fixing the conflicts, the description is present on top of the timeline and removed from the first history element as having it twice made no sense. With a bit of margin it looks very good now.